recipes/sklearn-quantile/meta.yaml

```diff
@@ -1,50 +0,0 @@
-{% set name = "sklearn-quantile" %}
```

Comment:
Please put this recipe back. The bot will handle it appropriately.

Reply:
I have restored this project, but this has created a merge conflict. Any ideas on how to solve this?

Reply:
There is still a change in the other recipe. Seems like just the line-ending. 
Once that‘s resolved it should be fine.

Reply:
That does indeed seem to solve the issue, thanks!